tests/ignite/engine/test_engine.py

```diff
@@ -794,7 +783,7 @@ def start_of_iteration_handler(engine):
     state = engine.run(data=[None] * num_iterations_per_epoch, max_epochs=max_epochs)
     # completes the iteration but doesn't increment counter (this happens just before a new iteration starts)
     assert state.iteration == num_iterations_per_epoch * (max_epochs - 1) + \
-        iteration_to_stop % num_iterations_per_epoch
+           iteration_to_stop % num_iterations_per_epoch
```

Comment:
Could you please restore previous indents and removed or added lines in this file. Thanks